lib/src/svg/xml_parsers.dart

```diff
@@ -125,19 +125,19 @@ PaintServer parseLinearGradient(XmlElement el) {
 
   parseStops(stops, colors, offsets);
 
+  final Matrix4 transform = parseTransform(getAttribute(el, 'gradientTransform', def: null));
+  
   return (Rect bounds) {
-    final Offset from = new Offset(
-      bounds.left + (bounds.width * x1),
-      bounds.left + (bounds.height * y1),
-    );
-    final Offset to = new Offset(
-      bounds.left + (bounds.width * x2),
-      bounds.left + (bounds.height * y2),
-    );
+    final from = Vector3(bounds.left + x1, bounds.top + y1, 0.0);
```

Comment:
This should be 
```
    Vector3 from = Vector3(bounds.left + (bounds.width * x1),
        bounds.top + (bounds.height * y1), 0.0);
   Vector3 to = Vector3(bounds.left + (bounds.width * x2),
        bounds.top + (bounds.height * y2), 0.0);
```

I can't remember exactly why the math needs to be that way, but without it the gradients in the samples/tests don't render properly.  You'll have to get rid of final because of something below....